lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
```

Comment:
Ok, so talking in person, we decide to continue as it is and I added on the PR description the deployment plan after this is merged. Which I will do right now.